Tests/Unit/Components/CategoryResolver/DefaultCategoryResolverTest.php

```diff
@@ -0,0 +1,49 @@
+<?php
+/**
+ * (c) shopware AG <info@shopware.com>
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace ShopwarePlugins\Connect\Tests\Unit\Components\CategoryResolver;
+
+use Shopware\Components\Model\ModelManager;
+use Shopware\CustomModels\Connect\ProductToRemoteCategoryRepository;
+use Shopware\CustomModels\Connect\RemoteCategoryRepository;
+use ShopwarePlugins\Connect\Components\CategoryResolver\DefaultCategoryResolver;
+use ShopwarePlugins\Connect\Tests\AbstractConnectUnitTest;
+
+class DefaultCategoryResolverTest extends AbstractConnectUnitTest
+{
+    /**
+     * @var DefaultCategoryResolver
+     */
+    private $defaultCategoryResolver;
+
+    /**
+     * @var  RemoteCategoryRepository
+     */
+    private $remoteCategoryRepository;
+
+    /**
+     * @before
+     */
+    public function prepareMocks()
+    {
+        $this->remoteCategoryRepository = $this->createMock(RemoteCategoryRepository::class);
+
+        $this->defaultCategoryResolver = new DefaultCategoryResolver(
+            $this->createMock(ModelManager::class),
+            $this->remoteCategoryRepository,
+            $this->createMock(ProductToRemoteCategoryRepository::class)
+        );
+    }
+
+    public function test_store_remote_categories_success()
+    {
+        $articleId = 4;
+        $categories = ['Category 1'];
+
+        $this->defaultCategoryResolver->storeRemoteCategories($categories, $articleId);
```

Comment:
Actually you need to verify which mock was called, which arguments were used ...
Right now this test only make a call and doesn't verify anything.

Reply:
A I thought it's enough to ensure that the test is running.. there are no callbacks. But I will add something like that fn xyz is called x times